### ISSUE TITLE
choice extension still work but append the result in another window of the same app

### ISSUE THREAD

**Issue by suredream** [espanso/espanso#1452]
**Describe the bug**
I wrote a match using the choice extension and trigger it in the input box in a web browser ( I have two windows of the browser).

The choice widget showed up, I made the selection but espanso just the other window, not the one I triggered it to input the results.

If I remove the second window and only working in one single window, espanso works fine.

However, I usually need two browser windows on the screens, run jupyter notebook in one and open web pages in another for reference.

**To Reproduce**

open two chrome windows, trigger a choice match in one and see how espanso works.

**Expected behavior**

espanso should stay in the same window to do the work.

**Setup information**
- OS: What OS are you using?  Mac OS 12.3
- Version: espanso 2.1.8

**Comment by smeech** [comment]
I agree, Espanso should replace text in the window from which it was triggered. 
Is this still an issue with the current release, or can we close it?

**Comment by aayushchhabra1999** [comment]
Still seems to be an issue for me. When I use multiple monitors, it just replaces the results to some random textbox on one of the apps. Doesn't even have to be same app from which trigger was initiated.

**Comment by tudinhlicel** [comment]
This is still reproducible for me on macOS Sonoma 14.5, espanso 2.4.0.

I have two displays, Chrome with several windows open, one per screen. Focus a Chrome window on the secondary display, type a choice/form trigger, pick a value. The widget opens on the primary display, and the result never expanded, also the focus lands in wrong window.

Single Chrome window works every time. Plain matches are unaffected.
